### ISSUE TITLE
A question about special task

### ISSUE THREAD

**Issue by Shuanglin-1126** [qhfan/RMT#39]
Thank you very much for your work, I have a question about the downstream task. When I use RMT and BiLSTM for action recognition tasks, the loss does not go down and the model does not converge. What is the problem with this, do you have any suggestions for this work?

**Comment by qhfan** [comment]
I also tried using RMT on the K400 dataset for action recognition, and its performance was quite impressive. I simply added a decomposition MaSA along the temporal dimension. This approach enables very stable training.

**Comment by Shuanglin-1126** [comment]
Thanks for the reply, the optimizer settings in your app are still AdamW, learning rate: 5e-4
